literature.bib

```diff
@@ -6284,6 +6330,25 @@ @article{TKB+:SCP14
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2013 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@book{JCB13,
+	title = {{Software Ecosystems: Analyzing and Managing Business Networks in the Software Industry}},
+	author = {Jansen, Slinger and Cusumano, Michael A and Brinkkemper, Sjaak},
+	year = 2013,
+	publisher = {Edward Elgar Publishing}
+}
+
+@article{MH:JSS13,
```

Comment:
publisher + address missing

Reply:
it is uncommon to specify publisher and address for journal articles

Reply:
Ok so I won't fix.

Reply:
That is one way to handle it, one the other hand one could argue that we have many fields that are uncommon (i.e. not filled in by dblp etc.). It is my opinion that the 5 minutes of extra efforts are worth it.

Reply:
I will add Elsevier as publisher but I have no idea where to find the address (within 5 minutes) as even the publishers themselves don't make this information public.

Reply:
The addresses are even filtered out when generating `literature-cleaned.bib`. So I don't see a reason here to spend more time searching for the needle in the haystack for this publication and all those to come in the next years.